sdk-jakarta/src/test/java/io/a2a/server/apps/jakarta/JakartaA2AServerTest.java

```diff
@@ -1,14 +1,76 @@
 package io.a2a.server.apps.jakarta;
 
-import jakarta.inject.Inject;
+
 
 import io.a2a.server.apps.common.AbstractA2AServerTest;
+import io.a2a.server.apps.common.AgentCardProducer;
+import io.a2a.server.apps.common.AgentExecutorProducer;
 import io.a2a.server.events.InMemoryQueueManager;
 import io.a2a.server.tasks.TaskStore;
-import io.quarkus.test.junit.QuarkusTest;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit5.container.annotation.ArquillianTest;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
 
-@QuarkusTest
+@ArquillianTest
+@ApplicationScoped
 public class JakartaA2AServerTest extends AbstractA2AServerTest {
+
+    public JakartaA2AServerTest() {
+        super(8080);
+    }
+
+    @Deployment
+    public static WebArchive createTestArchive() throws IOException {
+        final List<String> prefixes = List.of(
+                    "a2a-java-sdk-core",
```

Comment:
We'll need to update this to `a2a-java-sdk-spec` for https://github.com/a2aproject/a2a-java/pull/149. I'll add a note there after merging this one.